common/src/java/com/zimbra/common/gql/GqlConstants.java

```diff
@@ -53,10 +51,98 @@
     // end session constants
     public static final String CLEAR_COOKIES= "clearCookies";
 
+    // shared class constants
+    public static final String ADDRESS = "address";
+    public static final String ADDRESS_TYPE = "addressType";
+    public static final String ATTACHMENTS = "attachments";
+    public static final String CONTENT = "content";
+    public static final String CONTENT_ID = "contentId";
+    public static final String CONTENT_TYPE = "contentType";
+    public static final String FLAGS = "flags";
+    public static final String FOLDER_ID = "folderId";
+    public static final String FRAGMENT = "fragment";
+    public static final String HEADERS = "headers";
+    public static final String IDENTITY_ID = "identityId";
+    public static final String IN_REPLY_TO = "inReplyTo";
+    public static final String INVITE = "invite";
+    public static final String MESSAGE = "message";
+    public static final String MIME_PARTS = "mimeParts";
+    public static final String NAME = "name";
+    public static final String PERSONAL = "personal";
+    public static final String SUBJECT = "subject";
+
+    // message constants
+    public static final String CLASS_MESSAGE = "Message";
+    public static final String ATTACHMENT_ID = "attachmentId";
+    public static final String EMAILADDRESSES = "emailAddresses";
+    public static final String MIME_PART = "mimePart";
+    public static final String ORIGINAL_ID = "originalId";
+    public static final String REPLY_TYPE = "replyType";
+    public static final String TIMEZONES = "timezones";
+
+    // message specification constants
+    public static final String CLASS_MESSAGE_SPECIFICATIONS = "MsgSpec";
+    public static final String GQL_MESSAGE_SPECIFICATIONS = "messageSpecifications";
+    public static final String HIDE_IMAGES = "hideImages";
+    public static final String INLINED_TEXT_LENGTH = "inlinedTextLength";
+    public static final String MARK_MESSAGE_AS_READ = "markMessageAsRead";
+    public static final String MESSAGE_PART = "messagePart";
+    public static final String RECURRENCE_DATE_ID = "recurrenceDateId";
+    public static final String SHOW_GROUP_INFO = "showGroupInfo";
+    public static final String SHOW_IMAP_UID = "showImapUid";
```

Comment:
Use existing constant `INCLUDE_IMAP_UID`

Reply:
changed - in next commit